### PR TITLE
Base/Theme: Added a content area background colour variable.

### PR DESCRIPTION
**Pull request by EricDunsworth**
* Base:
  * Adds a $content-bg variable.
  * If a theme assigns a colour to $content-bg:
    * It's set as the background colour of the content area's container div in pages that contain secondary menus.
    * It's set as the background colour of the main element and the tabbed interface's active summary, unless a colour has been assigned to $main-bg.
* Theme:
  * Adjusts the $main-bg variable's declaration to re-use $breadcrumb-bg's value.
* Mostly addresses #7918.

**Note:** This will need to be merged alongside a few other upcoming PRs for other themes. I'll remove "[Do not merge]" from the subject once they're ready.

### CODE REVIEW

**Comment by EricDunsworth** [comment]
I've removed "[Do not merge]" since all related PRs are now up:
* wet-boew/theme-gcwu-fegc#244
* wet-boew/theme-base#134
* wet-boew/theme-ogpl#125
* GCWeb and the intranet theme are good as-is.

@LaurentGoderre FYI.

File: theme/secondary-menu/_base.scss [hunk 1]
```diff
@@ -7,8 +7,9 @@
 }
 
 #wb-sec {
-	margin-top: 20px;
```
Comment (LaurentGoderre) [inline theme/secondary-menu/_base.scss]: Why was this removed

Comment (EricDunsworth) [inline theme/secondary-menu/_base.scss]: I replaced ``margin-top`` with ``padding-top`` to allow that space to retain the left nav's background colour. It was part of my attempt to keep the [WET theme's secondary menu demo](https://wet-boew.github.io/v4.0-ci/theme/content-secmenu-en.html) looking the same as before.

Upon further observation, this PR doesn't seem to have turned out as well as I'd hoped. In the WET 4.0.22 screenshot below, the bottom of the left nav is light grey. But in both PR screenshots, it's white.

Screenshot of WET 4.0.22 (hasn't changed between 4.0.0-4.0.24):
wet-4 0 22-theme_secmenu_margin-top-20

Screenshot of this PR with ``margin-top`` (white background above the menu and white space below it):
wet-4 0 25pr-theme_secmenu_margin-top-20

Screenshot of this PR with ``padding-top`` (white space below the menu):
wet-4 0 25pr-theme_secmenu_padding-top-20

Comment (kodecount) [inline theme/secondary-menu/_base.scss]: Not having any background color behind _Content page_ could be a issue with people with poor vision.

**Comment by EricDunsworth** [comment]
I've set this back to "[Do not merge]" due to the issue shown in my review comment's screenshots. Probably going to need to rework my PRs to use separate colour variables for the container class and main element.

**Comment by EricDunsworth** [comment]
I've finally gotten around to resolving this PR's remaining issues, heavily tested it against other themes and have updated the related PRs I mentioned in an earlier comment.

Assuming no one has any concerns with the current state of this PR, I'd recommend merging it and all of its related PRs together. Once that's done, #7918 will probably need to be manually closed.

**Review by lucas-hay — APPROVED** [review]
Explained by @EricDunsworth at PR review meeting, and approved by members of meeting.
